### PR TITLE
Refactor/example

### PR DESCRIPTION
**Pull request by rewieer**
Refactored the example folder and fixed tests.
This will help us making tests for different types of Modals
Sorry for the quality of the gif, didn't find a fast tool to make it better (I don't know why but it's 2x slower on the gif 👎 )

Screen Recording 2019-11-10 at 07 48 10

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,7 +48,8 @@
   },
   "dependencies": {
     "prop-types": "^15.6.2",
-    "react-native-animatable": "1.3.3"
+    "react-native-animatable": "1.3.3",
+    "react-native-screens": "^1.0.0-alpha.23"
```
Comment (mmazzarolo) [inline package.json]: 🤔 wrong package?

Comment (rewieer) [reply package.json]: Followed the installation steps from react-navigation, which needs react-native-reanimated, react-native-screens and react-native-gesture-handler 😃

Comment (mmazzarolo) [reply package.json]: I mean, wrong `package.json`?

Comment (rewieer) [reply package.json]: Dammit yes, it's fixed

**Pull request merged**